templates/adherent/events.html.twig

```diff
@@ -54,7 +54,11 @@
                                     {{ event.beginAt|localizeddate('none', 'none', 'fr_FR', null, 'd MMMM Y') }} &mdash; à {{ event.cityName }}<br />
                                 </div>
                                 {% if event.type and event.isCitizenInitiative %}
-                                    <div class="text--body"><span class="text--bold">Initiative citoyenne</span> organisée par l'adhérent <span class="text--bold">{{ event.organizer.fullName|default('Anonyme') }}</span></div>
+                                    {% if event.organizer %}
```

Comment:
No need of `if` and even of `|default('Anonyme')`

Reply:
Shouldn't we keep consistency with the event list of the search template ?
https://github.com/EnMarche/en-marche.fr/blob/master/templates/search/_events.html.twig#L27

or should we refactor both ?

Reply:
got it. this check is useless on this page

Reply:
Sorry for this erroneous comment. 
This page lists events you are subscribed.
Your code is good !